docs/source/conf.py

```diff
@@ -87,7 +94,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "TraitsUI 4 User Manual"
+html_title = "TraitsUI 5 User Manual"
```

Comment:
Can we build this using the version number which is set somewhere earlier in the file?  That would prevent this particular error from ever occurring again.